super_resolution/main.py

```diff
@@ -28,14 +29,19 @@
     raise Exception("No GPU found, please run without --cuda")
 if not opt.mps and torch.backends.mps.is_available():
     raise Exception("Found mps device, please run with --mps to enable macOS GPU")
+if torch.xpu.is_available() and not opt.xpu:
+    raise Exception("WARNING: You have XPU device, to enable XPU run with --xpu.")
```

Comment:
Mimic mps and drop warning since this message is not a warning:
```suggestion
if not opt.xpu and torch.xpu.is_available():
    raise Exception("Found xpu device, please run with --xpu to enable it.")
```